lsr_role2collection.py

```diff
@@ -447,6 +446,12 @@ def convert_rolename(self, rolename, lsr_rolename=None):
                 )
         return new_name
 
+    def isstring(self, input):
+        try:
+            return isinstance(input, basename)
```

Comment:
basestring?

Reply:
Thank you for catching it! (I don't know why my test did not capture it... I guess to test it I should have a non-ascii task name...)

Reply:
Anyway, I've fixed my typo... :p

Reply:
> @richm, Thank you for catching it! ~(I don't know why my test did not capture it... I guess to test it I should have a non-ascii task name...)~
> 
> Reconsidering this change, I'm forcing to run lsr_role2collection.py with python3 even in the older env like rhel-7. That's said, we could simply replace the line 513 with `isinstance(task[module_name], str)`?

Yes, good idea.

> 
> Shebang of this script is `#!/usr/bin/env python3`. Could there be anything else we could claim this script only runs with python3?

Reply:
Thank you, @richm. Updated.